src/binary/ibc-setup/commands/ics20.spec.ts

```diff
@@ -107,32 +90,38 @@ destConnection: .+
   const srcConnectionIdMatch = /srcConnection: (?<connection>.+)/.exec(args[1]);
   const srcConnectionId = srcConnectionIdMatch?.groups?.connection;
   assert(srcConnectionId);
-  const srcClientMatch = /srcClient: (?<srcClient>.+)/.exec(args[1]);
-  const srcClient = srcClientMatch?.groups?.srcClient;
-  const nextConnectionWasm = await ibcClientWasm.query.ibc.connection.connection(
-    srcConnectionId
-  );
+  // const srcClientMatch = /srcClient: (?<srcClient>.+)/.exec(args[1]);
+  // const srcClient = srcClientMatch?.groups?.srcClient;
+  // const nextConnectionWasm = await ibcClientWasm.query.ibc.connection.connection(
```

Comment:
I see why you removed the client match (might as well just delete it not comment it out).

But why is `nextConnectionWasm` commented out? It should still work, right?

Reply:
I kept `nextConnection*` checks: https://github.com/confio/ts-relayer/pull/94/files#diff-5bb34d24bff296d595d81dafe53eb04885ccbf17a096785882bdc8be02806a17R116-R117

but only as simple asserts since I cannot capture `clientId` from the test anymore.